src/borg/crypto/key.py

```diff
@@ -375,7 +375,7 @@ def detect(cls, repository, manifest_data):
                     raise PasswordRetriesExceeded
         else:
             if not key.load(target, passphrase):
-                raise PassphraseWrong
+                raise PassphraseWrong(passphrase)
```

Comment:
please read the code above, you only covered env passphrase.

Reply:
I see that `Passphrase.env_passphrase()` handles `BORG_PASSPHRASE`, `BORG_PASSCOMMAND`, and `BORG_PASSPHRASE_FD`. Are you suggesting I need to adjust how `PassphraseWrong` is raised in these cases, or is there another specific issue you had in mind?

Reply:
The goal of this is to be able to be helpful with "passphrase wrong" issues, no matter how the pp is entered.

If somebody enters the passphrase on the console and does not manage to enter a correct one, it will raise PasswordRetriesExceeded - but that is just a special case of PassphraseWrong.

Reply:
Thank you for the clarification! I've updated the code to ensure that debugging information is displayed after every failed attempt, no matter how the passphrase is entered.

If you'd like, I can further streamline the debugging output for interactive retries by skipping environment variable details `(e.g., BORG_PASSPHRASE, BORG_PASSCOMMAND, etc.)`, as those may not always be relevant in an interactive context.